tests/kuttl/test-replica-scaledown/03-pods.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdApp
+metadata:
+  name: puptoo
+  namespace: test-replica-scaledown
+spec:
+  envName: test-replica-scaledown
+  deployments:
+  - name: processor
+    minReplicas: 0
+    podSpec:
+      image: quay.io/psav/clowder-hello
+---
+apiVersion: apps/v1
```

Comment:
Remove this Deployment from 03-pods -- since the intent of the test is to ensure lowering minReplicas on the ClowdApp to '0' works, don't change the Deployment at this point. The assert is good for this test.